ports/hiredis/portfile.cmake

```
@@ -33,11 +33,29 @@ vcpkg_copy_pdbs()
 
 vcpkg_fixup_pkgconfig()
 
-file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include")
 vcpkg_cmake_config_fixup()
 if("ssl" IN_LIST FEATURES)
     vcpkg_cmake_config_fixup(PACKAGE_NAME hiredis_ssl CONFIG_PATH share/hiredis_ssl)
 endif()
 
+vcpkg_replace_string("${CURRENT_PACKAGES_DIR}/include/hiredis/hiredis.h"
+[[typedef long long ssize_t;
+#define _SSIZE_T_ /* for compatibility with libuv */]]
+[[#if !defined(_SSIZE_T_) && !defined(_SSIZE_T_DEFINED)
+typedef long long ssize_t;
+#define _SSIZE_T_ /* for compatibility with libuv */
+#endif]]
+)
+vcpkg_replace_string("${CURRENT_PACKAGES_DIR}/include/hiredis/sds.h"
+[[typedef long long ssize_t;
+#define SSIZE_MAX (LLONG_MAX >> 1)]]
+[[#if !defined(_SSIZE_T_) && !defined(_SSIZE_T_DEFINED)
+typedef long long ssize_t;
+#define SSIZE_MAX (LLONG_MAX >> 1)
+#endif]]
+)
```

```suggestion
vcpkg_replace_string("${CURRENT_PACKAGES_DIR}/include/hiredis/hiredis.h"
[[typedef long long ssize_t;
#define _SSIZE_T_ /* for compatibility with libuv */]]
[[typedef intptr_t ssize_t;]]
)
vcpkg_replace_string("${CURRENT_PACKAGES_DIR}/include/hiredis/sds.h"
[[typedef long long ssize_t;
#define SSIZE_MAX (LLONG_MAX >> 1)]]
[[typedef intptr_t ssize_t;
#define SSIZE_MAX INTPTR_MAX]]
)
```

This may also require including `<stdint.h>` in those files though and may be clearer as a patch.